### PR TITLE
Added/Updated files for 16.1

### PR DESCRIPTION
**Pull request by traghavendra**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# custom cinder-volume container - having python-3parclient
+FROM registry.redhat.io/rhosp-rhel8/openstack-cinder-volume:16.1
```
Comment (sneharai4) [inline Dockerfile:2]: Does the registry change for this?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+MAINTAINER HPE
+
+LABEL name="rhosp16/openstack-cinder-volume-hpe" \
```
Comment (sneharai4) [inline Dockerfile]: Should the label be rhosp16.1 instead?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      maintainer="sneha.rai@hpe.com" \
+      vendor="HPE" \
+      release="16.1" \
+      summary="Red Hat OpenStack Platform 16.1 cinder-volume HPE plugin" \
+      description="Cinder plugin for HPE 3PAR"
```
Comment (sneharai4) [inline Dockerfile]: Cinder plugin for HPE 3PAR and Primera?

**Review by sneharai4 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (traghavendra) [reply Dockerfile:2]: Yes. i verified from RH page. Around 8 days back, cinder-volume 16.1 was created.

File: Dockerfile (hunk 2 above)
Comment (traghavendra) [reply Dockerfile]: ok. will update to 16.1

File: Dockerfile (hunk 3 above)
Comment (traghavendra) [reply Dockerfile]: ok. will add Primera.

**Review by sneharai4 — APPROVED** [review]

**Pull request merged**